ui-components/src/icons/Icons.tsx

```diff
@@ -0,0 +1,541 @@
+import * as React from "react"
+
+export interface IconProps {
+  fill?: string
+}
+
+export const Application = (props: IconProps) => {
+  return (
+    <svg viewBox={"0 0 24 32"} fill={props.fill ?? "#000000"}>
```

Comment:
I think would be better to use `currentColor` instead of `#000000`, what do you think? (noticed multiple places where we can change it)